src/tasks/start/start-backend.ts

```diff
@@ -103,13 +106,16 @@ export async function startBackend(
 
   // Call preInit hook.
   if (hooks && hooks.preInit) {
-    await hooks.preInit({ proxy, _experimentalAppInstaller }, bre)
+    await hooks.preInit(
+      { proxy, _experimentalAppInstaller, log: logHook('preInit') },
+      bre
+    )
   }
 
   // Call getInitParams hook.
   let proxyInitParams: any[] = []
   if (hooks && hooks.getInitParams) {
-    const params = await hooks.getInitParams({}, bre)
+    const params = await hooks.getInitParams({ log: logHook('initP') }, bre)
```

Comment:
Might as well use `'initParams'` to make it more clear

Reply:
Fixed by https://github.com/aragon/buidler-aragon/pull/115